package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hmrc-frontend",
-  "version": "1.31.0",
+  "version": "1.33.0",
```

Comment:
@matthewmascord I think if you run npm install it will bump this in package-lock.json too

Reply:
+1 Should be done now @oscarduignan